source/LibUISharp/src/LibUISharp/Drawing/FontFeatures.cs

```diff
@@ -52,11 +52,16 @@ public void Remove(string feature)
             Remove(chars[0], chars[1], chars[2], chars[3]);
         }
 
-        public int TryGetValue(byte a, byte b, byte c, byte d, out uint value) => LibuiLibrary.uiOpenTypeFeaturesGet(Handle.DangerousGetHandle(), a, b, c, d, out value);
+        public int TryGetValue(byte a, byte b, byte c, byte d, out long value)
+        {
+            var result = LibuiLibrary.uiOpenTypeFeaturesGet(Handle.DangerousGetHandle(), a, b, c, d, out var uintValue);
```

Comment:
I also don't really like using `var`. Just my preference.

(I know, I need to get a `.editorconfig` for this repo, but I don't know much about that either)